### ISSUE TITLE
why coco-stuff 256x256 LostGAN-v2 fid=30.82 in your paper while 42.55 in LostGAN-v2's paper

### ISSUE THREAD

**Issue by ZGCTroy** [wb-finalking/PLGAN#2]
Thanks for your impressive work and i have some questions below:
1. why coco-stuff 256x256 LostGAN-v2 fid=30.82 in your paper while 42.55 in LostGAN-v2's paper? what is the number of generated samples for evaluation? Is the one-time of validation dataset (about 2048 or 3072 in Layout2im or 3097 in LostGAN-v2) or five-times of validation dataset? 
2.  what is the reference ground truth images for evaluation. Is the one-time of validation dataset (about 2048 or 3072 in Layout2im or 3097 in LostGAN-v2)?
3.  the code for evaluation from saved imgs, pytorch version or tensorflow version of fid evaluator?

**Comment by wb-finalking** [comment]
Sorry for the delay in replaying.
1. We use pretrained model from LostGAN-v2 and make evaluations on the five-times of validation dataset. And we find the metrics are all the same as the LostGAN-v2's paper expect this FID number. So we show the better one in our paper.
2. You can refer to the evaluation code [here](https://github.com/WillSuen/LostGANs/issues/3).

**Comment by ZGCTroy** [comment]
Thanks for your kindly reply !!!
One more question: Could you please share the config or training parameters of your resnet101  for calculating CAS (classification accuracy score) at resolution 32x32. Although we use the evaluation github code your provided https://github.com/hysts/pytorch_image_classification, we can't reproduce the results, such as 51.04 for real images and your (CAL2I [6]+PLG) 32.33. There is only widely used resnet-110 for cifar10/100, and resnet-101 for imagenet.
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/28867789/186370453-6850aaba-9d52-41e5-a1c8-adb576957e82.png">
